src/routes/blog/developer-journey/timeleap-protocol.tiramisu

```diff
@@ -0,0 +1,143 @@
+title { Timeleap: The Communication Protocol }
+
+So, we've talked about Timeleap giving you the tools to create a distributed
+computation network. In this article, we are connecting the dots: Timeleap, Sia,
```

Comment:
> So, we've talked about Timeleap giving you the tools to create a distributed computation network.

Where did we do that? Perhaps a link would be useful here. Also, don't start the article with "So,"